content/review.rst

```diff
@@ -93,6 +96,10 @@ For the person requesting a review
 * Check the **Reviewer's check list** since those are the things that a
   reviewer will check for sure.
 
+* Before submitting a ticket for review, check that you have documented your
+  work accordingly, like in a wiki page, or if you pushed a PR, list
```

Comment:
As a sysadmin, you will still need to touch the private wiki... for ip addresses, service names ...etc

If possible, I would like to have the styleguide in such a way so that it will describe the "philosophy"  of the review process. rather than being a step by step (for dummies) guide for how to do a review using GitHub.

-------

Also, as part of the whole styleguide, we should include information and best practices based on our remote / async work ... and to help people understand what is specific to remote work, why we do one thing and not another.